src/psyclone/tests/domain/lfric/kern_call_arg_list_test.py

```diff
@@ -482,3 +482,48 @@ def test_kerncallarglist_scalar_literal(fortran_writer):
         create_arg_list.scalar(args[3])
     assert ("Unexpected literal expression 'invalid' in scalar() when "
             "processing kernel 'testkern_qr_code'" in str(err.value))
+
+
+def test_indirect_dofmap_apply(fortran_writer):
+    '''Test the indirect dofmap apply function.
```

Comment:
I'm not sure what this means. Does a dofmap have an 'apply' function?

Reply:
I have absolutely no idea :blush: Apply removed.